src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Square extends Figure {
```

Comment:
The class Square should not start with an empty line. Please remove the redundant empty line at the beginning of the class definition.